vexcl/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+
+FILE(GLOB_RECURSE MyCSources *.hpp )
+add_library(vex ${MyCSources})
+
+SET_TARGET_PROPERTIES(vex PROPERTIES LINKER_LANGUAGE CXX)
```

Comment:
I don't understand this change. There is no library to build here, vexcl is header-only. One only need to add vexcl root directory to include paths with a compiler switch.